File: LinkedList/Simple/simpleLL.cpp
#include<iostream>
using namespace std;

class Node{
    public:
      int data;
      Node* next;
      Node(int val){
          this->data=val;
          this->next=NULL;
      }
};

void printLinkedList(Node *n){
   while(n!=NULL){
       cout<<n->data<<" ";
       n=n->next;
   }
}

int main(){
    Node* head=new Node(10);
    Node* second=new Node(20);
    Node* third=new Node(30);

    head->next=second;
    second->next=third;

    printLinkedList(head);
}

File: LinkedList/Simple/simpleLL.js
let head;

class Node {
  constructor(val) {
    this.data = val;
    this.next = null;
  }
}

function printList(head) {
  while (head != null) {
    console.log(head.data);
    head = head.next;
  }
}

head = new Node(1);
let second = new Node(2);
let third = new Node(3);

head.next = second;
second.next = third;

printList(head);
